From 29939f8e4c06bf8a1c5d32b95fce367e245d3508 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Tue, 8 Sep 2020 13:24:03 +0800
Subject: [PATCH] hypervisor: dont build pre_build for target

Execute pre_build_check during hypervisor build is causing error :

| make: /data/yocto/poky/build-acrn/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/2.1-r0/build//hypervisor/hv_prebuild_check.out: Command not found

Build and execute as native.

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 hypervisor/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/hypervisor/Makefile b/hypervisor/Makefile
index e28dc63d..19c3d36e 100644
--- a/hypervisor/Makefile
+++ b/hypervisor/Makefile
@@ -395,9 +395,6 @@ install-debug: $(HV_OBJDIR)/$(HV_FILE).map $(HV_OBJDIR)/$(HV_FILE).out
 
 .PHONY: pre_build
 pre_build: $(HV_OBJDIR)/$(HV_CONFIG_H)
-	@echo "Start pre-build static check ..."
-	$(MAKE) -C $(PRE_BUILD_DIR) BOARD=$(BOARD) SCENARIO=$(SCENARIO) TARGET_DIR=$(TARGET_DIR)
-	@$(HV_OBJDIR)/hv_prebuild_check.out
 	@echo "generate the binary of ACPI tables for pre-launched VMs ..."
 	python3 ../misc/acrn-config/acpi_gen/bin_gen.py --board $(BOARD) --scenario $(SCENARIO) --asl $(TARGET_DIR) --out $(HV_OBJDIR)/acpi
 
-- 
2.17.1

